inference-engine/tests/functional/plugin/myriad/subgraph_tests/nonzero_broadcast.cpp

```diff
@@ -19,6 +22,13 @@ class NonZero_Broadcast : public testing::WithParamInterface<BroadcastExplicitTe
                           public LayerTestsUtils::LayerTestsCommon {
 protected:
     void SetUp() override {
+        SetRefMode(LayerTestsUtils::RefMode::CONSTANT_FOLDING);
+        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
+        // DISABLE_REORDER is needed for Myriad2 cases
+        if (!CommonTestUtils::vpu::CheckMyriadX()) {
```

Comment:
Nitpick: I'd prefer to have a function for detecting MY2 - can be fixed in a separate request

Reply:
Done

Reply:
Oh, I sorry for misleading you - I meant 2 different functions, actually.
But for now it's ok.